test_suite/cloud/test_aws.py

```diff
@@ -367,20 +367,22 @@ def test_max_cstate_is_configured_in_cmdline(self, host):
                         f'{line} must not be specified in AMIs that are not SAP'
 
     @pytest.mark.run_on(['all'])
-    @pytest.mark.jira_skip(["CLOUDX-994"])
     def test_hostkey_permissions(self, host):
         """
         Check that ssh files permission set are correct.
         BugZilla 2013644
+        Ensure permissions are aligned with a distro and release version
+        ClOUDX-994
```

Comment:
Code changes look good to me.
Gives me some time to nit-pick ;) 
```suggestion
        CLOUDX-994
```